### DISCUSSION TITLE
[Discussion] Under equivalent conditions, the performance of Dubbo 3 in terms of QPS (Queries Per Second) and RT (Response Time) is not as good as Dubbo 2

### DISCUSSION THREAD

**Discussion by roadToit** [apache/dubbo#14021] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dubbo/issues?q=is%3Aissue) and found no similar issues.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Dubbo Version

Dubbo 3.1.11
Dubbo 2.6.12
OpenJdk 11
Linux

### Steps to reproduce this issue

Based on the provided load testing results, where three types of Dubbo interfaces were tested under both Dubbo 2 and Dubbo 3:

A blank Dubbo interface with no logic.
A Dubbo interface returning a POJO object.
A Dubbo interface returning a list of POJOs.
The load testing results indicate the following:

The QPS (Queries Per Second) of interfaces based on Dubbo 3 decreased significantly compared to those based on Dubbo 2.
The RT (Response Time) metrics (Max/TP9999/TP999/TP99/avg) of interfaces based on Dubbo 3 showed varying degrees of increase, indicating slower performance compared to interfaces based on Dubbo 2.
Considering the above load testing results, the question arises whether it is still necessary to upgrade from Dubbo 2 to Dubbo 3.

### What you expected to happen

I expected Dubbo 3 to show improvements in terms of QPS and RT compared to Dubbo 2, but the actual load testing results did not reflect this. 
I am quite confused.

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
1. Please test on 3.2.12
2. Can you please provide the statistics about your benchmark?
3. Can you please provide the Flame Graph of profiling
